builder/remotecontext/urlutil/urlutil_test.go

```diff
@@ -17,6 +17,7 @@ var (
 	}
 	invalidGitUrls = []string{
 		"http://github.com/docker/docker.git:#branch",
+		"https://github.com/docker/dgit",
```

Comment:
We might have to change this example if we revisit https://github.com/docker/cli/issues/1738 (since this really _should_ be a valid Git URL), but these don't have to be real repos so at that point it could change to something like `https://example.com/dgit` and still be sufficient for the case it's verifying. :+1: